.github/workflows/test-package.yml

```
@@ -20,17 +20,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [dev]
-        version: [latest]
-        include:
-          - sdk: stable
-            version: 2.12.0
+        sdk: [2.12.0, dev]
```

```suggestion
        sdk: [dev]
```

We don't want to have the formatting check on both SDK.

I had been configuring to do analyzer on both, but I think I've convinced myself since that it is not worth it..